### DISCUSSION TITLE
Private List

### DISCUSSION THREAD

**Discussion by m-primo** [community/community#8626] [Other Feature Feedback, Questions, & Ideas]
A feature to make a list private.

**Comment by AbhimanyuAryan** [comment]
Want private lists as well. Doesn't make sense to tell your IDEAS/TODO publicly before they are in execution/done

**Reply by llagerlof** [reply]
I mean, it's basic privacy. The user doesn't need a specific reason to keep his data private.

**Reply by m-primo** [reply]
totally agree

**Reply by httpanand** [reply]
Looks like the star lists are already private .. ?

**Reply by m-primo** [reply]
> Looks like the star lists are already private .. ?

No, it's not.. I tried to access from another account and i saw it.

**Reply by httpanand** [reply]
oh k .. Yesterday it was private ..  Now they have made it public .. I think they're frequently updating those .. :)

**Reply by AbhimanyuAryan** [reply]
please consider upvoting this so it's pushed up on priority @httpanand

**Reply by Berachem** [reply]
agree

**Reply by Nooknick7899** [reply]
Ok

**Reply by Greekkerjonny** [reply]
> Want private lists as well. Doesn't make sense to tell your IDEAS/TODO publicly before they are in execution/done

**Reply by Agrament** [reply]
Akaoe

**Reply by Agrament** [reply]
aandziaaa28@gmail.com

**Reply by bmgzz** [reply]
https://github.com/orgs/community/discussions/38693#discussioncomment-4127687

**Comment by AbhimanyuAryan** [comment]
Also how about add feature to only show "Star Lists" to sponsors and not making make it completely `Private`. Educators can compile interesting projects and students can sponsor to access the list. This will also boost adoption of Github `Sponsors`. It will add more value to `Github Sponsors`

@bailey

**Reply by Nooknick7899** [reply]
Very good

**Reply by Lumitex** [reply]
OK.

**Reply by Phuchong0259** [reply]
> Very good 

OK

**Reply by Phuchong0259** [reply]
OK

**Comment by ghost** [comment]
Everyone, I was reading everyone's comment here - What has been suggested and thought about this private list proposal. Through the gui -  graphical user interface, feature list and user story. I hope to clear up a few things that everyone here wants or thinks about the private list idea.

#### Features
1. List - public type. This list is for everyone to see. At the moment is a standard list that Github offers and everyone has access, this list is public.
2. List - private type. Only you will have access. This is a list that everyone wants here. This list is private, meaning no one has access. If the user allows them access, this list is no longer a private list and becomes a public list.
3. List - unlisted type. This allows you to select people who can access information from the list. This includes sponsors or people who follow you. I left this type of list as optional, because can be something to be taken into account. But, I don't know if it makes sense in this topic.

#### User stories
- As a user, want my list to be public, if I want as a user and allow
- As a user, want my list to be private, if I want as a user and allow
- As a user, want my list to be accessed by people who follow me or only by someone sponsoring me, if I want as a user and allow

#### image to repository(private/public/fork)
![image](https://user-images.githubusercontent.com/95204402/147844031-99fd9876-94e9-4f64-b2f4-269e0cef5778.png)

#### image to user profile
![image](https://user-images.githubusercontent.com/95204402/147842588-29ba7d69-fe2d-4e46-a186-3919c24db1f2.png)

Note: I've read all the opinions, suggestions and comments so I thought I'd make a general comment on that. I hope to help and I apologize if you have any errors, if you have any errors, be sure to comment that I will make a correction

**Reply by Vmrbee03** [reply]
I need my children

**Reply by Lumitex** [reply]
The same

**Reply by mjraha592** [reply]
> I need 
> I > I need my children
 >
 Hello, please make the list completely private and personal on my platform. Thank you and your colleagues

**Reply by mjraha592** [reply]
> Everyone, I was reading everyone's comment here - What has been suggested and thought about this private list proposal. Through the gui -  graphical user interface, feature list and user story. I hope to clear up a few things that everyone here wants or thinks about the private list idea.
> 
> #### Features
> 1. List - public type. This list is for everyone to see. At the moment is a standard list that Github offers and everyone has access, this list is public.
> 2. List - private type. Only you will have access. This is a list that everyone wants here. This list is private, meaning no one has access. If the user allows them access, this list is no longer a private list and becomes a public list.
> 3. List - unlisted type. This allows you to select people who can access information from the list. This includes sponsors or people who follow you. I left this type of list as optional, because can be something to be taken into account. But, I don't know if it makes sense in this topic.
> 
> #### User stories
> - As a user, want my list to be public, if I want as a user and allow
> - As a user, want my list to be private, if I want as a user and allow
> - As a user, want my list to be accessed by people who follow me or only by someone sponsoring me, if I want as a user and allow
> 
> #### image to repository(private/public/fork)
> ![image](https://user-images.githubusercontent.com/95204402/147844031-99fd9876-94e9-4f64-b2f4-269e0cef5778.png)
> 
> #### image to user profile
> ![image](https://user-images.githubusercontent.com/95204402/147842588-29ba7d69-fe2d-4e46-a186-3919c24db1f2.png)
> 
> Note: I've read all the opinions, suggestions and comments so I thought I'd make a general comment on that. I hope to help and I apologize if you have any errors, if you have any errors, be sure to comment that I will make a correction

> I need my children
 >
 Hello, please make the list completely private and personal on my platform. Thank you and your colleagues

**Reply by josh583050** [reply]
> Everyone, I was reading everyone's comment here - What has been suggested and thought about this private list proposal. Through the gui -  graphical user interface, feature list and user story. I hope to clear up a few things that everyone here wants or thinks about the private list idea.
> 
> #### Features
> 1. List - public type. This list is for everyone to see. At the moment is a standard list that Github offers and everyone has access, this list is public.
> 2. List - private type. Only you will have access. This is a list that everyone wants here. This list is private, meaning no one has access. If the user allows them access, this list is no longer a private list and becomes a public list.
> 3. List - unlisted type. This allows you to select people who can access information from the list. This includes sponsors or people who follow you. I left this type of list as optional, because can be something to be taken into account. But, I don't know if it makes sense in this topic.
> 
> #### User stories
> - As a user, want my list to be public, if I want as a user and allow
> - As a user, want my list to be private, if I want as a user and allow
> - As a user, want my list to be accessed by people who follow me or only by someone sponsoring me, if I want as a user and allow
> 
> #### image to repository(private/public/fork)
> ![image](https://user-images.githubusercontent.com/95204402/147844031-99fd9876-94e9-4f64-b2f4-269e0cef5778.png)
> 
> #### image to user profile
> ![image](https://user-images.githubusercontent.com/95204402/147842588-29ba7d69-fe2d-4e46-a186-3919c24db1f2.png)
> 
> Note: I've read all the opinions, suggestions and comments so I thought I'd make a general comment on that. I hope to help and I apologize if you have any errors, if you have any errors, be sure to comment that I will make a correction

**Reply by bmgzz** [reply]
https://drive.google.com/file/d/17jjLeT01uFNrrWvQNtuy7F4MzvdnkvvJ/view?usp=drivesdk&disco=AAABQbDYv1o

**Comment by bailey** [comment]
Hi everyone! Thanks so much for sharing this feedback. We hear you and are definitely considering how we could roll out private lists in the future. In the meantime, feel free to share here how private lists could better your lists experience. ✨

**Reply by EnochianD** [reply]
Anytime thank you🕊️🤗👍😁

**Reply by ezdub** [reply]
Some repos you hold closer to your chest like a good poker hand. A small moat. 
Currently I’m using browser bookmarks 🤢 as private lists but it’s too disconnected from the rest of the GitHub experience like returning my starred repos in searches and how seeing a repo I’ve previously starred visually reminds me I’ve had prior interest when I land in the same spot again.

**Reply by adityamenon** [reply]
Hi Bailey, would love to have my lists set to private most of the time, as it's a way to organize my own thoughts and don't need these to be public. A compromise where the Stars themselves are visible, but the List Names are not visible, would also be acceptable to me (although complete privacy would be best).

**Reply by Zj220** [reply]
> Hi everyone! Thanks so much for sharing this feedback. We hear you and are definitely considering how we could roll out private lists in the future. In the meantime, feel free to share here how private lists could better your lists experience. ✨

**Reply by Lumitex** [reply]
.

**Reply by Lumitex** [reply]
Thanks you.

**Reply by pataquets** [reply]
Sort of a bookmarking device, which might have several use cases with required privacy (research, keeping your stack, future projects, ideas, etc)

**Reply by EngineerIT14** [reply]
Easy to answer, I want organization in my repo and I am using lists to do it, I want a private list to group all my private repos so it can be easier for me to navigate. As it is now, I am creating lists and I can see the private repos on them BUT people visiting my profile see empty list and also reading the list description. I want this to be private so my repo can be maintained clean!

**Reply by mjraha592** [reply]
Trust vallet assix

**Reply by mjraha592** [reply]
> Sort of a bookmarking device, which might have several use cases with required privacy (research, keeping your stack, future projects, ideas, etc)

Ok

**Reply by Dandreholley123** [reply]
Ok

**Reply by geekley** [reply]
Yeah, I just want to bookmark repos on GitHub without a 2nd thought to worry about privacy. For example:
- Without being listed under "Stargazers" for that repo
- Without all my private bookmarks being viewable by others on my profile
- Would be even better to also (possibly behind a profile setting) make my "default" starring private, so I could only make it public if I explicitly want to, i.e. by adding to a public list.

I'd even go further and make private the default, since :star: doesn't really imply publicity like stuff such as :+1: and arguably :heart: do.

Stack Exchange did something similar.
They had a "mark favorite" feature which was public (worse, in a obscure way; it wasn't obvious that it was public).
Then they renamed it to "bookmarks" with made the problem worse (as the name heavily implies it's private).
Then finally fixed it with a "saves" feature which is lists like this, but they changed it to private migrating all previous bookmarks.

Currently I have to workaround this by editing a private gist with a bunch of links. Which is a lame workaround.

**Reply by Hamidsalri** [reply]
https://sdr.hu/openwebrx

**Reply by Hamidsalri** [reply]
![Uploading dumpstate.txt …]()

**Reply by Hamidsalri** [reply]
{
  "ignores": [
    "@lavamoat/allow-scripts",
    "@lavamoat/preinstall-always-fail",
    "@metamask/auto-changelog",
    "@types/*",
    "prettier-plugin-packagejson",
    "ts-node",
    "typedoc"
  ]
}

**Reply by 4ff0rdsUn1c0rns** [reply]
> Hi Bailey, would love to have my lists set to private most of the time, as it's a way to organize my own thoughts and don't need these to be public. A compromise where the Stars themselves are visible, but the List Names are not visible, would also be acceptable to me (although complete privacy would be best).

The issue I battle is that it's centered around a FOSS community...
Private organization of lists ... Do we display our organization processes as well... ie: public stars with option of private lists.

I've just realized lists are public; and, I've been looking for export options which keep the lists.

Thought I found some the other day...
Then .. searching for .. unstar in bulk...

Private Stars ... Maybe a CLI system... So they're actually private on the users own system.

**Comment by 317580** [comment]
Hddgjhfdhddf

**Comment by 317580** [comment]
Mhamad .syrea.@317580

**Comment by 317580** [comment]
> Hddgjhfdhddf

Dddddd

**Comment by Jawjomnissan** [comment]
> A feature to make a list private.
_[]()_

**Reply by nepalauziamas** [reply]
Dds

**Reply by Danlachapelle** [reply]
> > A feature to make a list private.
> _[]()_
>

**Reply by mjraha592** [reply]
> > A feature to make a list private.
> _[]()_
> 

Mjraha592

**Comment by nepalauziamas** [comment]
Sss

**Reply by Lumitex** [reply]
222

**Comment by AmandaP87** [comment]
> Everyone, I was reading everyone's comment here - What has been suggested and thought about this private list proposal. Through the gui -  graphical user interface, feature list and user story. I hope to clear up a few things that everyone here wants or thinks about the private list idea.
> 
> #### Features
> 1. List - public type. This list is for everyone to see. At the moment is a standard list that Github offers and everyone has access, this list is public.
> 2. List - private type. Only you will have access. This is a list that everyone wants here. This list is private, meaning no one has access. If the user allows them access, this list is no longer a private list and becomes a public list.
> 3. List - unlisted type. This allows you to select people who can access information from the list. This includes sponsors or people who follow you. I left this type of list as optional, because can be something to be taken into account. But, I don't know if it makes sense in this topic.
> 
> #### User stories
> - As a user, want my list to be public, if I want as a user and allow
> - As a user, want my list to be private, if I want as a user and allow
> - As a user, want my list to be accessed by people who follow me or only by someone sponsoring me, if I want as a user and allow
> 
> #### image to repository(private/public/fork)
> ![image](https://user-images.githubusercontent.com/95204402/147844031-99fd9876-94e9-4f64-b2f4-269e0cef5778.png)
> 
> #### image to user profile
> ![image](https://user-images.githubusercontent.com/95204402/147842588-29ba7d69-fe2d-4e46-a186-3919c24db1f2.png)
> 
> Note: I've read all the opinions, suggestions and comments so I thought I'd make a general comment on that. I hope to help and I apologize if you have any errors, if you have any errors, be sure to comment that I will make a correction

- [ ] # ****

**Comment by wemow** [comment]
Just created a new thread about this, it should have been implemented since a while imo.

**Reply by Lumitex** [reply]
.

**Reply by Hamidsalri** [reply]
https://github.com/community/community/discussions/8626#discussioncomment-3945969

**Reply by bmgzz** [reply]
_https://github.com/orgs/community/discussions/8626#discussioncomment-10122995_

**Reply by Marverlinnck** [reply]
@adityamenon

**Comment by Dmutro777** [comment]
пн, 24 окт. 2022 г., 2:05 forty ***@***.***>:

**Comment by iambmg007** [comment]
> A feature to make a list private.

Who's using Chrome?
With Chrome profiles you can separate all your Chrome stuff. Create profiles for friends and family, or split between work and fun. Sounds good to me. Yes, I will. Sounds good to me, exclaimed! Hell 🌈?
OohH...Just Because. iambmg.

**Reply by wemow** [reply]
A bit off topic no? 
I might be wrong, you are implying that you should use multiple github accounts just for the same of a list?

**Reply by Lumitex** [reply]
.

**Comment by iambmg007** [comment]
- [ ] [2733](657833)

**Comment by iambmg007** [comment]
Who's using Chrome?
With Chrome profiles you can separate all your Chrome stuff. Create profiles for friends and family, or split between work and fun. Sounds good to me. Yes, I will. Sounds good to me, exclaimed! Hell 🌈?
OohH...Just Because. In.

**Comment by Lumitex** [comment]
.

**Comment by Lumitex** [comment]
Yes

**Comment by Lumitex** [comment]
Yes

**Comment by ooahz** [comment]
2023/5/11 It remains unimplemented

**Reply by ToMe-code** [reply]
**DTF**

**Reply by Laneraverfa** [reply]
Hugo

**Reply by Laneraverfa** [reply]
[]()```
# > - [ ] ~~~~
```

**Reply by Vanche38** [reply]
🤣

**Comment by QuietHubb** [comment]
Love

**Comment by ghost** [comment]
https://workingcopy.app/git/#commit=b1301d8cf3d1a10b62f649ffff2fdc09d9c07d06&repo=git@github.com:Baramos20/Converted---StaceyLynnAnderson-InteliusReport.git https://workingcopy.app/git/#commit=b1301d8cf3d1a10b62f649ffff2fdc09d9c07d06&repo=git@github.com:Baramos20/Converted---StaceyLynnAnderson-InteliusReport.git

**Reply by Sammyreal1** [reply]
Approved and exchange

**Comment by DavidMichaeldomainglobal10365** [comment]
> A feature to make a list private.

Excellence

**Reply by Sulthanmh** [reply]
Thread

**Comment by waruwu93** [comment]
Hdbbdnxm dia mam

**Comment by Littlewarrior3** [comment]
_[](https://github.com/orgs/community/discussions/8626)_

**Comment by nitre** [comment]
Lmao clinic

**Comment by Ikhanaghafazeli** [comment]
Ikhanaghafazeli4@outlook.com****

**Comment by Vasyanhik** [comment]
> A feature to make a list private.

**Comment by mohon8156** [comment]
ওকে ফাইন সবাই সবার মত উওর দেওয়ার জন্য আর মন্তব্য করার জন্য।

**Comment by Niteshwe** [comment]
Hii

**Reply by ghost** [reply]
> Hii 
>

**Comment by jykae** [comment]
My 2 cents for this feature discussion:
* Public / Private option with major priority
* ability to share list with other Github users as nice-to-have feature

**Comment by aaronh197** [comment]
Copy privacy policys

**Comment by Hamidsalri** [comment]
> Hi everyone! Thanks so much for sharing this feedback. We hear you and are definitely considering how we could roll out private lists in the future. In the meantime, feel free to share here how private lists could better your lists experience. ✨

**Comment by Robedee** [comment]
> A feature to make a list private.

**Comment by Robedee** [comment]
> > Hi everyone! Thanks so much for sharing this feedback. We hear you and are definitely considering how we could roll out private lists in the future. In the meantime, feel free to share here how private lists could better your lists experience. ✨ 
> 
>

**Comment by Coool** [comment]
Any progress, news?

**Comment by Nina530** [comment]
> > A feature to make a list private.
> 
> 


**

**

**Comment by shoaib9804725** [comment]
Ok

**Comment by DrakkoFire** [comment]
Any news on this? This feature would be very useful...

**Reply by OSHORDESMOND** [reply]
![Screenshot_20241109-142643.jpg](https://github.com/user-attachments/assets/298cf98c-3c63-4229-b662-215efe501bd8)

**Comment by Nordtom** [comment]
> > A feature to make a list private.
> 
> Excellence 

>money For @Nordtom

**Comment by Yehuda-Bergstein** [comment]
Any progress?

**Reply by MP3Martin** [reply]
Looks like it's been added

![Screenshot_20260218_230049_Iceraven](https://github.com/user-attachments/assets/71b42414-0f59-4b52-b7a7-eab4cb955e2c)

**Reply by Yehuda-Bergstein** [reply]
Thanks for the update!

**Reply by ftrippel** [reply]
Still visible as starred project under your public profile. The list itself is private though

**Reply by mootari** [reply]
Unfortunately private lists aren't marked as private in the UI. You have to edit them to discover which ones are private vs public.

**Reply by EngineerIT14** [reply]
FINALLY 🤘

**Comment by nlozkina19-crypto** [comment]
Hey everyone! I just released vector-zero-compute (a high-performance C++/Python crypto core). I'm looking for any feedback on the code or architecture. Drop your repo link in the replies, and I will check out your project and leave a star in return! ⭐ https://github.com/nlozkina19-crypto/vector-zero-compute

**Comment by zorenkonte** [comment]
Heads up for anyone confused like I was, GitHub did add per-list privacy. There's a "Private" checkbox in the Edit list modal now.

Still annoying though. No bulk option, no documented API. If you've got a bunch of lists to flip, I had Claude generate a script for me using the (undocumented) updateUserList GraphQL mutation and tested it myself. Sets all your lists private at once via gh CLI. https://gist.github.com/zorenkonte/a46041b77c335d6f1bd27b5c1b099214

Figured it might save someone the clicking. Still hoping this gets proper API support at some point.
